README.md

```diff
@@ -1,5 +1,7 @@
 # DynaFlow Launcher
 
+Dynawflow Launcher (DFL) depends on the [core dynawo libraries of Dynawo](https://github.com/dynawo/dynawo)
+
```

Comment:
While we are modifying this file, could you please add a definition of a deployed DYNAWO?

Reply:
I just add "depends on the deployed installation..." ?